client/src/components/TicketForm/TicketForm.jsx

```diff
@@ -84,8 +83,12 @@ class TicketForm extends Component {
 		}
 
 		if (this.state.description.value && this.state.location.value && this.state.contact.value) {
+			this.props.getCurrentUser();
+			if (! this.props.user.currentUser){
```

Comment:
There will never be a time when a user can see the ticket dashboard but not be logged in. Dustin's next task is to implement access control for this.